src/kleinanzeigen_bot/resources/categories.yaml

```diff
@@ -0,0 +1,591 @@
+    Wii: 161/279/wii
+    Xbox: 161/279/xbox
+  Notebooks: 161/278
+  PC-Zubeh�r & Software:
+    Drucker & Scanner: 161/225/drucker_scanner
+    Festplatten & Laufwerke: 161/225/festplatten_laufwerke
+    Geh�use: 161/225/gehaeuse
+    Grafikkarten: 161/225/grafikkarten
+    Kabel & Adapter: 161/225/kabel_adapter
+    Mainboards: 161/225/mainboards
+    Monitore: 161/225/monitore
+    Multimedia: 161/225/multimedia
+    Netzwerk & Modem: 161/225/netzwerk_modem
+    Prozessoren / CPUs: 161/225/prozessor_cpu
+    Software: 161/225/software
+    Speicher: 161/225/speicher
+    Tastatur & Maus: 161/225/tastatur_maus
+    Weiteres PC-Zubeh�r: 161/225/sonstiges
+  PCs: 161/228
+  TV & Video:
+    DVD-Player & Recorder: 161/175/dvdplayer_recorder
+    Fernseher: 161/175/fernseher
+    TV-Receiver: 161/175/tv_receiver
+    Weitere TV & Video: 161/175/weitere
+  Tablets & Reader:
+    Kindle: 161/285/kindle
+    Samsung Tablets: 161/285/samsung_tablets
+    Weitere Tablets & Reader: 161/285/weitere
+    iPad: 161/285/ipad
+  Videospiele:
+    DS(i)- & PSP Spiele: 161/227/dsi_psp
+    Nintendo Spiele: 161/227/nintendo
+    PC Spiele: 161/227/pc_spiele
+    PlayStation Spiele: 161/227/playstation
+    Weitere Videospiele: 161/227/sonstige
+    Wii Spiele: 161/227/wii
+    Xbox Spiele: 161/227/xbox
+  Weitere Elektronik: 161/168
+Familie, Kind & Baby:
+  Altenpflege: 17/236
+  Baby- & Kinderkleidung:
+    Accessoires: 17/22/accessoires
+    Bademode: 17/22/bademode
+    Hemden: 17/22/hemden
+    Hosen & Jeans: 17/22/hosen_jeans
+    Jacken & M�ntel: 17/22/jacken_mantel
+    Kleider & R�cke: 17/22/kleider_roecke
+    Kleidungspakete: 17/22/kleidungspakete
+    Pullover & Strickjacken: 17/22/pullover_strickjacken
+    Shirts & Tops: 17/22/shirts_tops
+    Sportbekleidung: 17/22/sportbekleidung
+    Weitere Baby- & Kinderkleidung: 17/22/sonstiges
+    W�sche: 17/22/wasche
+  Baby- & Kinderschuhe:
+    Badeschuhe: 17/19/badeschuhe
+    Ballerinas: 17/19/ballerinas
+    Halb- & Schn�rschuhe: 17/19/halb_schnuerschuhe
+    Hausschuhe: 17/19/hausschuhe
+    Outdoor & Wanderschuhe: 17/19/outdoor_wanderschuhe
+    Sandalen: 17/19/sandalen
+    Sneaker & Sportschuhe: 17/19/sneaker_sportschuhe
+    Stiefel & Stiefeletten: 17/19/stiefel_stiefeletten
+    Weitere Schuhe: 17/19/sonstiges
+  Baby-Ausstattung: 17/258
+  Babyschalen & Kindersitze: 17/21
+  Babysitter/-in & Kinderbetreuung: 17/237
+  Kinderwagen & Buggys: 17/25
+  Kinderzimmerm�bel:
+    Betten & Wiegen: 17/20/betten_wiegen
+    Hochst�hle & Laufst�lle: 17/20/hochstuehle_laufstaelle
+    Schr�nke & Kommoden: 17/20/schraenke_kommoden
+    Weitere Kinderzimmerm�bel: 17/20/sonstige
+    Wickeltische & Zubeh�r: 17/20/wickeltische_zubehoer
+    Wippen & Schaukeln: 17/20/wippen_schaukeln
+  Spielzeug:
+    Action- & Spielfiguren: 17/23/actionfiguren
+    Babyspielzeug: 17/23/babyspielzeug
+    Barbie & Co: 17/23/barbie
+    Dreirad & Co: 17/23/dreirad
+    Gesellschaftsspiele: 17/23/gesellschaftsspiele

```

Comment:
So I'm a little bit confused.
I replaced the category in the yaml to `17/23/babyspielzeug`
But this category was not set.

So I published by hand in the same category and downloaded the add. There was the category set to `17l463/23l463`
When I triy to publish this ad, this category also not set.

What I'm doing wrong?

Reply:
So far as I see is the problem while saving/downloading an ad. It saves the category like `category: 161l463/225l463` instead of `161/225/festplatten_laufwerke`
Do we need a mapping table or something? Or are you able to download and (re)publish the downloaded ads?